src/lib389/lib389/replica.py

```diff
@@ -1666,6 +1685,22 @@ def begin_task_ldif2cl(self):
         """
         self.replace('nsds5task', 'ldif2cl')
 
+    def task_finished(self):
```

Comment:
Is this ever called outside of a testing context?

Reply:
It is used by the CLI, and now in the CI tests as well.